decrypt/decrypt-sd-submission

```diff
@@ -22,8 +22,8 @@ with tarfile.open(input) as tar:
   # given a malicious tarball
   tar.extractall(tmpdir)
 
-# everything in the archive should be zips, as created by SD
-# let's unzip those here
+# some files may be zips, some may be .gpg
+# let's unzip the zip files here to produce .gpg files
```

Comment:
There's a problem here... when we encounter zip files, we unzip in into `tmpdir + "/extracted/"`, which creates that directory if needed. Then, on line 54, we create a new tarball from the files in`tmpdir + "/extracted/"`.

The "bare" gpg files don't make it into .`.../extracted/` (and indeed `.../extracted/` isn't created if we're only dealing with bare gpg files) so the tarball creation on line 54 fails.

I suppose we could make the `.../extracted` dir by hand and move the bare gpg files into place there?

Reply:
It's sort of a drag because the only indication of the source name we have is the name of the `.gpg` file... for the example i'm working with now, that's `2-underhanded warrior-doc.gz-1`

Reply:
Good call @joshuathayer, I'll add a block to handle this